api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -34,6 +34,50 @@ const (
 	PrivateDNSZoneModeNone string = "None"
 )
 
+// UpgradeChannel - For more information see setting the AKS cluster auto-upgrade channel [https://docs.microsoft.com/azure/aks/upgrade-cluster#set-auto-upgrade-channel].
+type UpgradeChannel string
+
+const (
+	// UpgradeChannelNodeImage - Automatically upgrade the node image to the latest version available. Consider using nodeOSUpgradeChannel
+	// instead as that allows you to configure node OS patching separate from Kubernetes version patching.
```

Comment:
```suggestion
	// instead as that allows you to configure node OS patching separate from Kubernetes version patching.
	
```

Please add a newline in between the declaration as well for consistency